### PR TITLE
Fix/spacing proof of incident

### PR DESCRIPTION
**Pull request by Milanpanta**
- Fix spacing of input boxes for proof of incident
- Change unit for maximum bond and your bond

### CODE REVIEW

File: src/components/UI/molecules/pools/bond/splitted-card.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const SplitedDetail = ({
       </h6>
       <p
         className={classNames(
-          "text-lg font-normal font-numbers whitespace-nowrap",
+          "text-lg font-normal font-numbers whitespace-normal",
```
Comment (flashburst) [inline src/components/UI/molecules/pools/bond/splitted-card.jsx:23]:
```suggestion
          "text-lg font-normal font-numbers",
```
Remove redundant class name

File: src/components/pages/pools/bond/index.jsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const BondPage = () => {
     },
     {
       title: "Maximum Bond",
-      value: `${bondInfo?.max_bond} NPM`,
+      value: `${bondInfo?.max_bond} NPM USDC LP`,
```
Comment (flashburst) [inline src/components/pages/pools/bond/index.jsx:33]: This should be "NPM-USDC LP"

**Review by flashburst — CHANGES_REQUESTED** [review]

File: src/components/UI/molecules/pools/bond/splitted-card.jsx (hunk 1 above)
Comment (Milanpanta) [reply src/components/UI/molecules/pools/bond/splitted-card.jsx]: Done.

File: src/components/pages/pools/bond/index.jsx (hunk 2 above)
Comment (Milanpanta) [reply src/components/pages/pools/bond/index.jsx]: Done

**Review by flashburst — APPROVED** [review]

**Pull request merged**